crates/symbolicator/src/services/download/filesystem.rs

```diff
@@ -63,6 +64,38 @@ impl FilesystemDownloader {
         &self,
         file_source: FilesystemRemoteDif,
         dest: PathBuf,
+    ) -> Result<DownloadStatus, DownloadError> {
+        let retries =
+            future_utils::retry(|| self.download_source_once(file_source.clone(), dest.clone()));
```

Comment:
Are filesystem failures really worth retrying?  The chances of them improving are very slim I'd say.  Though it also doesn't do any harm I guess.

Reply:
Good point: I figured it'd be better to just make sure these are all consistent instead of making more exceptions for filesystem downloads. It does add some amount of complexity to their implementation, but I think the tradeoff is fine for now.